CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # cuML 0.8.0 (Date TBD)
 
 ## New Features
+<<<<<<< HEAD
```

Comment:
seems like the conflict has not been resolved correctly!? Such things need to be removed.